build.gradle

```diff
@@ -268,6 +269,22 @@ dependencies {
     }
 }
 
+configurations {
+    // due to a change in newer loom, loader isn't remapped
+    // except some mods pull in an older version of loader that *is* remapped
+    // so it has to be excluded explicitly.
+
+    modRuntimeOnly.exclude group: "net.fabricmc", module: "fabric-loader"
```

Comment:
Is loader still there when running the game with this?

Reply:
yes, this just excludes it transitively. loader is in the modImplementation configuration, not modRuntimeOnly.